src/java.desktop/macosx/classes/sun/lwawt/macosx/CPlatformWindow.java

```diff
@@ -187,6 +191,16 @@ static boolean IS(final int bits, final int mask) {
         return (bits & mask) != 0;
     }
 
+    static {
+        AccessController.doPrivileged(
+                (PrivilegedAction<Object>) () -> {
+                    allowMacOSAutomaticWindowTabbing = Boolean.parseBoolean(
+                            System.getProperty("jdk.allowMacOSTabbedWindows"));
```

Comment:
Just a thing to discuss, probably we can extract the string of the property to where other similar properties are stored already.